### PR TITLE
De-removes recapitation defibrillation

### PR DESCRIPTION
**Pull request by mistyLuminescence**
**WIP**

TL;DR version: A while ago, if someone was decapitated, it'd sometimes be possible to take their head and reattach it to their body, and then defibrillate them. This was removed in #4264 ; this PR is effectively a revert of that PR. To be discussed in [this forum thread](https://forum.ss13polaris.com/viewtopic.php?f=14&t=500).

- [X] Get recapitation defibrillation working
- [ ] Add a complex (many steps) surgery
- [ ] Other tweaks (for both patient and doc)

### CODE REVIEW

**Comment by Wickedtemp** [comment]
Anewbe said to open up a PR if we disagreed with it, so... That's what this is.

**Comment by Curt5** [comment]
In favor of re-adding recapitations. It's one of those times where 'It's not a bug, it's a feature' is actually true.

**Comment by Mechoid** [comment]
I'm going to point out, removing the can_defib is not advisable to do this, if you're going to be touching this. I'll be posting my own thoughts in the thread in a moment.

**Comment by mistyLuminescence** [comment]
There's probably a more elegant way of fixing it, yeah. Maybe setting can_defib to 1 when the head gets reattached? You still can't defib someone without a head, at least, because they're missing a vital organ (brain). Thoughts on how to implement it would definitely be appreciated.

**Comment by Mechoid** [comment]
I've posted in the thread, and I'm not personally comfortable discussing my own proposal for making a re-capitation defibbable, due to and more than what I've noted in the thread.

**Comment by Atermonera** [comment]
Controversial because this will probably take a lengthy discussion, regardless of outcome.
[Forum Thread](https://forum.ss13polaris.com/viewtopic.php?f=14&t=500)

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -15,7 +15,6 @@
 	var/spit_name = null 				//String
 	var/last_spit = 0 					//Timestamp.
 
-	var/can_defib = 1					//Horrible damage (like beheadings) will prevent defibbing organics.
```
Comment (Belsima) [inline code/modules/mob/living/carbon/human/human.dm]: wow removing this is an absolutely terrible idea please do not

**Comment by Arokha** [comment]
It looks like a lot of the comments in favor here and in the thread are 'because this thing is hard to do, I want to be able to do it and get a reward', which is a bit nonsensical. You could say the same about virtually anything not possible in the game if it's hard (really, just if 'you think' it's hard), and if that's your single factor arguing in favor of it, then it's probably not actually worth doing.

Another thing I noticed is that people seem to be confusing the OOC difficulty and IC difficulty in doing this in their arguments. I see several people saying 'no, this isn't that easy' but are referring specifically to the OOC game mechanical difficulty of it and likelihood of being able to accomplish it, while others are saying 'too easy' referring to the in-universe seeming ease of attaching a limb requiring virtually no effort.

That aside, I do agree with Mechoid's comments in the thread. The tools available in surgery are insufficient to even vaguely represent the in-universe difficulty one would encounter when reattaching limbs. Maybe it should require the IMS to do, plus drugs, plus more steps or something, who knows.

**Comment by mistyLuminescence** [comment]
> TL;DR: Keep it, because:
* It's cool.
* It requires lots of skill and luck, so doesn't happen all the time.
* It's easy to shut down if the antag takes the time to disposal/space/husk/maul/hide the body.
* Removing it, but keeping it for synths, widens the power gap between FBP and organic.
* It's not unrealistic, given the setting and technology level.
* It rewards medical players for putting in effort to save the patient.
* When it's relevant, it doesn't take anything away from the server, but adds a lot.

From the original PR. It's not so much as "we should be able to do it because it's hard", it's "we should be able to do it because it's awesome and rewarding, but it should be hard so it doesn't happen all the time" (it already was hard before it was removed). Adding a complex surgery would go a long way towards making this more thematic, more fun, and more awesome.

**Comment by mistyLuminescence** [comment]
There's something even better coming Soon ™️ - stay tuned and keep an eye on the forum thread. You'll know it when you see it. Keeping the forum thread open 'cause the topic's still going, it just won't be implemented with this PR specifically.
